.meta/_partials/fields/_request_options.toml.erb

```diff
@@ -77,3 +77,42 @@ recommended that you do not lower this value below the service's internal timeou
 as this could create orphaned requests, pile on retries, and result in \
 duplicate data downstream.\
 """
+
+[<%= namespace %>.request.children.decrease_ratio]
+type = "float"
+common = true
```

Comment:
I'd be fine with setting these as uncommon. I feel like ideally most people would not need to set them, but they'll be very useful for when they do.

Reply:
Agreed